### ISSUE TITLE
static parsing error with change of regime

### ISSUE THREAD

**Issue by christophsax** [christophsax/seasonal#205]
For example:
```
air.m <- seas(AirPassengers, transform.function = "log", x11="", forecast.maxlead=24, 
          regression.aictest=NULL, outlier=NULL, arima.model="( 0 1 1)", 
          regression.variables = c("seasonal/1955.1/", "td"), 
          slidingspans = NULL) 
air.static <- static(air.m,test=FALSE)
```
gives you the following in air.static:
```
seas(x = AirPassengers, transform.function = "log", x11 = "", 
   forecast.maxlead = 24, arima.model = "( 0 1 1)", 
   regression.variables = c("seasonal/r", 
                            "before", "1955.Jan/", "td"), 
   slidingspans = NULL, regression.aictest = NULL, 
   outlier = NULL)
```
This gives errors when you try to use the static seas object with functions like spc:

```
> spc(eval(air.static))
Error: X-13 run failed

Errors:
- Not a valid date
- Expected a date not "before"
- Regression variable name "before" not found
- Expected regression variable name or ")" but found "1955."
- Argument name "estimate" not found
- Expected NAME=VALUE or NAME=(LIST) not "{"
>
```

**Comment by christophsax** [comment]
### Problem

The underlying problem is the content of the `.mdl` file, which looks mixed with some text for theses regime changes:

```r
 regression{
  variables=(
   td/ for before 1955.Jan/
     )
  b=(
           -0.5410284591E-03
           -0.9463792658E-02
           -0.9840078589E-02
            0.9182325245E-02
           -0.8878602543E-02
            0.1498740740E-01
           -0.3548262970E-02
            0.6796760559E-02
            0.1010064903E-01
           -0.1496189472E-01
            0.8987247176E-02
            0.3852592153E-02
     )
 }

 arima{model=
  ( 0 1 1)
   ma  =(
           -0.3401302070E+00
   )
 }
```

Is there a better source to pick up this information?

### Workaround

I added a workaround that tries to filter out that text and works for most cases

```r

static(seas(x = AirPassengers, regression.variables = "td/1955.Jan/"))

static(seas(x = AirPassengers, regression.variables = "td//1955.Jan/"))

static(seas(x = AirPassengers, regression.variables = "td/1955.Jan//"))

static(seas(x = AirPassengers, regression.variables = c("td", "td/1955.Jan//")))

static(seas(x = AirPassengers, regression.variables = c("td", "td//1955.Jan/"), regression.aictest = NULL))

static(seas(x = AirPassengers, regression.variables = c("seasonal", "td//1955.Jan/"), regression.aictest = NULL))

static(seas(x = AirPassengers, regression.variables = c("seasonal", "td/1955.Jan/"), regression.aictest = NULL))

static(seas(x = AirPassengers, regression.variables = c("seasonal", "td/1955.Jan//"), regression.aictest = NULL))

static(seas(x = AirPassengers, regression.variables = c("td1coef", "td1coef/1955.Jan//"), regression.aictest = NULL))

static(seas(x = AirPassengers, regression.variables = c("td1coef", "td1coef/1955.Jan//"), regression.aictest = NULL))

static(seas(x = AirPassengers, regression.variables = c("seasonal", "td/1955.Jan/"), regression.aictest = NULL))

static(seas(x = AirPassengers, regression.variables = c("seasonal", "lpyear/1955.Jan/"), regression.aictest = NULL))

static(seas(x = AirPassengers, regression.variables = c("seasonal", "lom/1955.Jan/"), regression.aictest = NULL))
```

However, for `tdstock1coef`, it still fails, but I guess we leave this as it is for the moment.

```r
static(seas(x = AirPassengers, regression.variables = c("tdstock1coef[1]/1955.Jan/"), regression.aictest = NULL))
```

**Comment by christophsax** [comment]
workaround implemented in `1.6.1`. To be revisited if `.mdl` file is modified by Census.
